GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -2503,34 +2503,34 @@ private static Image GetItemImage(SubmoduleInfo info)
             return info.IsDirty ? Resources.IconSubmoduleDirty : Resources.Modified;
         }
 
-        private static void GetSubmoduleStatusAsync(SubmoduleInfo info, CancellationToken cancelToken)
+        private static async Task GetSubmoduleStatusAsync(SubmoduleInfo info, CancellationToken cancelToken)
```

Comment:
:bulb: We don't use `CancellationToken` throughout the code yet, but in code that does the parameter name is typically `cancellationToken`